imaging_transcriptomics/inputs.py

```diff
@@ -89,7 +88,9 @@ def load_gene_expression():
     :return: numpy array with the gene expression data.
     """
     logger.debug("Loading gene_expression data.")
-    expression_file_path = Path(__file__).resolve().parent / "data" / "gene_expression_data.csv"
+    expression_file_path = Path(
+        __file__).resolve().parent.parent / "data" / "gene_expression_data.csv"
```

Comment:
The path should be `.parent/ "data" / "gene_expression_data.csv"` since the location of the data folder has changed